code/modules/food_and_drinks/food.dm

```diff
@@ -1,19 +1,67 @@
 ////////////////////////////////////////////////////////////////////////////////
 /// Food.
 ////////////////////////////////////////////////////////////////////////////////
+GLOBAL_LIST_INIT(temp_stabilizers, typecacheof(list(
+	/obj/structure/closet/crate/secure/hydroponics,
+	/obj/structure/closet/crate/freezer,
+	/obj/structure/closet/crate/hydroponics,
+	/obj/structure/closet/secure_closet/freezer
+	)))
+
 /obj/item/reagent_containers/food
 	possible_transfer_amounts = list()
 	volume = 50	//Sets the default container amount for all food items.
 	container_type = INJECTABLE_1
 	resistance_flags = FLAMMABLE
 	var/foodtype = NONE
 	var/last_check_time
+	var/cares_about_temperature = FALSE
+	var/starting_temp = 200 // Farenheight
```

Comment:
we do temps in kelvin

Reply:
Are you gonna use ozs and cups too in recipes? How about some love for us EU wannabe chefs?

Reply:
got a point. When I talked to chefs, they used Celsius. Kelvin is at least neutral in terms of nationality and makes sense code-wise...

Reply:
fuckng fahremjoegh is bad degenerate system pls use celsisus

Reply:
you're not using kelvin because it's good for cooking you're using it because it's the codebase standard, convert to whatever value you want to for display, add vars that convert between, but all calculation must be in kelvin

Reply:
Goof this is awful coding standards even for you.

Reply:
You don't know what you're talking about.

Reply:
"Let me completely change the paradigm of this object property because MY features are special snowflake and deserve to express that by setting horrible, confusing precedents."

I seriously doubt the maintainers will let this through until you change it to kelvin, but if they do, I'll be happy to make a patch to convert it to kelvin because this is fucking stupid. Especially because your goal is to remove emergent gameplay because spooky powergamer boogeyman wants to fight antags instead of dick around playing watered down cooking mama.

Reply:
are you high